### DISCUSSION TITLE
Add a system tray wingpanel indicator or implement tray in an existing indicator

### DISCUSSION THREAD

**Discussion by GabrieleFazio** [elementary/triage#376] [Ideas]
elementary OS currently lacks any system tray functionality, and while that makes it look gorgeous, it isn't very practical for non-elementary apps (for example Discord or KDE Connect).

I am aware of the existance of [MvBonin's wingpanel-community-indicators](https://github.com/MvBonin/wingpanel-community-indicators) as I use it, but I'd like to see something similar implemented in upstream elementary OS.

**Comment by secondlnl** [comment]
elementary OS intentionally doesn't support a system tray [see this wingpanel issues#96 comment by Cassidy](https://github.com/elementary/wingpanel/issues/96#issuecomment-401407354).

**Comment by danirabbit** [comment]
Yeah sorry gonna close this because the situation hasn’t really changed. There’s still no cross-platform specification for a concept of application indicators in freedesktop. It seems like there’s some more work going into the background apps portal for apps to convey more information about their background status, but it doesn’t make sense for us to implement a dead end API that everyone is trying to get away from
